components/profile-about-tab/profile-about-tab.jsx

```diff
@@ -7,16 +7,23 @@ class ProfileAboutTab extends React.Component {
     super(props);
   }
 
+  renderLongBio() {
+    if (!this.props.longBio) return null;
+
+    return this.props.longBio.split(`\n`).map(line => {
```

Comment:
I guess we can't just replace the `\n` with `<br>` because we're escaping html?

Reply:
Yup and `<br>` isn't a markdown syntax either.